app/views/non_hmis_assessments/_index.haml

```diff
@@ -0,0 +1,60 @@
+- default_assessment = @non_hmis_client.assessment_type.constantize.new.default?
+%section
+  .c-card.c-card--flush
+    .c-card__content.p-0
+      .o-pre-table-actions.o-pre-table-actions--light
+        %h2.o-pre-table-actions__heading Assessments
+        - if @non_hmis_client.editable_by?(current_user)
+          .o-pre-table-actions__group.ml-auto
+            - if default_assessment
+              - new_assessment_url = polymorphic_path([:new_assessment, @non_hmis_client])
+            - else
+              - new_assessment_url = new_polymorphic_path([@non_hmis_client, :non_hmis_assessment])
+            = link_to new_assessment_url, class: 'btn btn-sm btn-secondary' do
+              %span.icon-plus
+              Add New Assessment
+      - if @assessments.any?
+        %table.table
+          %thead
+            %tr
+              %td Date Taken
+              - if can_manage_identified_clients? || Config.get(:identified_client_assessment) != 'IdentifiedCovidPathwaysAssessment'
```

Comment:
I don't think this is correct, shouldn't it infer from the assessment based on type? (or maybe from the controller)